Render.py

```diff
@@ -209,6 +210,18 @@ def Activated(self):
             import ImageGui
             ImageGui.open(img)
 
+class CameraCommand:
+
+    "Create a Camera object"
```

Comment:
`"""Triple quotes"""`
`"""Create a Camera object"""`

Reply:
You're right! However, actually, Render.py needs a comprehensive linting/refactoring not limited to that, so I take your point but I'll fix it (among others) when I do such a linting job. It is the next task on my roadmap!